### PR TITLE
Update README with development installation instructions

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready/Hold | [Ticket](https://pi-top.atlassian.net/browse/OS-1373) |

#### Main changes
Update README with development installation instructions

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: packages/pitop/README.rst [hunk 1]
```diff
@@ -126,57 +126,84 @@ Check out the `CLI`_ chapter of the documentation for ideas on how to get starte
...
-To make changes to the SDK you'll want to install it from source with the
-documentation and test dependencies:
+On ARM devices, such as the Raspberry Pi, you need to include pi-top's repository to meet all the dependencies:
 
 .. code-block:: bash
 
-    git clone https://github.com/pi-top/pi-top-Python-SDK.git
-    cd pi-top-Python-SDK
-    pip3 install -e ".[doc,test]"
+  pip3 install pitop --extra-index-url=https://packagecloud.io/pi-top/pypi/pypi/simple
```
[the remark this answers is not in the document]
Comment (angusjfw) [reply packages/pitop/README.rst:148]: `https://packages.pi-top.com/pypi/pypi/simple` - should work already, assuming pip will follow redirects like apt does

File: README.rst [hunk 2]
```diff
@@ -161,6 +161,42 @@ You can also install a pitop subpackage directly in case you don't need the whol
...
+To install the SDK in development mode, the subpackages need to be handled in a particular order, given
+the dependencies between them. Also, the :code:`--extra-index-url` parameter is added to install the :code:`onnxruntime` package
+instead of building it from scratch for some architectures.
+
+
+.. code-block:: bash
+
+  git clone https://github.com/pi-top/pi-top-Python-SDK.git
+  cd pi-top-Python-SDK
+  pip3 install \
```
Comment (angusjfw) [inline README.rst:178]: it would be best to just reference the [dev-install.sh](https://github.com/pi-top/pi-top-Python-SDK/pull/580/files#diff-1b9652fe4dcfae05f0b0fc24810862e42fdae6e7bca2417857122b98e37846d7) script here so that people know to use that rather than paste all this.

The information above about what it does could stay, but just add something like "We've put this into the script dev-install.sh."

File: packages/pitop/README.rst [hunk 3]
```diff
@@ -126,57 +126,84 @@ Check out the `CLI`_ chapter of the documentation for ideas on how to get starte
 Installation
 ------------
 
+OS
```
Comment (angusjfw) [inline packages/pitop/README.rst:129]: I don't see why we should have a full repeat of the main readme in the main subpackage readme here, unless it's getting installed from here or it's for the pypi page or something along those lines?

I think it would be best to just have a short description like the other subpackages. eg "This is the top level package that collects together the subpackages, plus a few extras like 'labs'."

Maybe you could reference the main readme for people who are looking for that. (If you do this maybe put a full github url rather than just relative path for scenarios like pypi page)

Comment (jcapona) [reply packages/pitop/README.rst]: yeah, I didn't remove it since it's what someone will see when going to the `pitop` package in pypi. I'll keep the images and update the description to explain what that package in particular does.

**Review by angusjfw — APPROVED** [review]

File: README.rst [hunk 4]
```diff
@@ -72,17 +72,14 @@ When this library reaches v1.0.0, we will aim to maintain backwards-compatibilit
 About
 -----
 
-This SDK aims to provide an easy-to-use framework for managing a pi-top. It includes a Python 3 package (`pitop`),
-with several modules for interfacing with a range of pi-top devices and peripherals It also contains CLI utilities,
+This SDK aims to provide an easy-to-use framework for managing a pi-top. It includes several Python 3 packages and
+modules for interfacing with a range of pi-top devices and peripherals It also contains CLI utilities,
 to interact with your pi-top using the terminal.
```
Comment (olivierwilkinson) [inline README.rst:77]: I know this wasn't introduced by this PR but would be good to get it in at the same time. :)
```suggestion
modules for interfacing with a range of pi-top devices and peripherals.
It also contains CLI utilities for interacting with your pi-top using the terminal.
```

File: README.rst [hunk 5]
```diff
@@ -161,6 +158,23 @@ You can also install a pitop subpackage directly in case you don't need the whol
...
 
+-----------
+Development
+-----------
+
+
+To install the SDK in development mode, the subpackages need to be handled in a particular order, given
+the dependencies between them. Also, the :code:`--extra-index-url` parameter is added to install the :code:`onnxruntime` package
+instead of building it from scratch for some architectures. You can use the :code:`dev-install.sh` script to install the SDK
+in development mode.
```
Comment (olivierwilkinson) [inline README.rst:169]: I think it would be helpful to put the command to run at the beginning, the details can come after and are mostly relevent to people where the script does not work for their usecase.

```suggestion
To install the SDK in development mode run the script `dev-install.sh` in the root of the repo:

.. code-block:: bash
  ./dev-install.sh

The script installs the subpackages in a particular order, given
the dependencies between them. Also, the :code:`--extra-index-url` parameter is added to install the :code:`onnxruntime` package
instead of building it from scratch for some architectures.
```

**Review by olivierwilkinson — COMMENTED** [review]
I have a couple small comments but overall looks good 👍

File: README.rst [hunk 6]
```diff
@@ -161,6 +158,27 @@ You can also install a pitop subpackage directly in case you don't need the whol
...
 
+-----------
+Development
+-----------
+
+
+To install the SDK in development mode run the script `dev-install.sh` in the root of the repo:
+
+.. code-block:: bash
+  ./dev-install.sh
```
Comment (jcapona) [inline README.rst:169]:
```suggestion
.. code-block:: bash

  ./dev-install.sh
```

File: README.rst [hunk 7]
```diff
@@ -161,6 +158,28 @@ You can also install a pitop subpackage directly in case you don't need the whol
 
   pip3 install pitop.pma
 
+-----------
+Development
+-----------
+
+
+To install the SDK in development mode run the script `dev-install.sh` in the root of the repo:
```
Comment (jcapona) [inline README.rst:166]:
```suggestion
To install the SDK in development mode run the script :code:`dev-install.sh` in the root of the repo:
```
